QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -652,49 +826,64 @@ ElasticProblem<dim>::ElasticProblem()
 
   // Just use Cole+ (2013) elastic moduli.
   // Supplemental document, section S2.
```

Comment:
Please add a comment explaining why all are divided by 1000.0. I presume this is because they use GPa=1 and we use TPa = 1.